test/actions/asks/ask_channel_status_for_yesterday.test.ts

```
@@ -1,22 +1,378 @@
+const mockGetStatsMessage =
+  jest.fn<(channelId: string, stats: AsksChannelStatsResult) => string>();
+const mockScheduleAskChannelsCrons =
+  jest.fn<
+    (
+      client: any,
+      teams: Team[],
+      channelIdKey: string,
+      channelNameKey: string,
+      cronKey: string,
+      command: string,
+      callback: Function,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)